roles/postfix/tasks/main.yml

```diff
@@ -0,0 +1,84 @@
+---
+# tasks file for roles/postfix
+- name: Postfix | add mailname
+  ansible.builtin.copy:
+    dest: "/etc/mailname"
+    content: |
+      "{{ ansible_facts['hostname'] }}"
+    force: false
+    owner: root
+    group: root
+    mode: "0644"
+
+- name: Postfix | Install Postfix
+  ansible.builtin.apt:
+    name: postfix
+    state: present
+    update_cache: true
+
+- name: Postfix | create networks file
+  ansible.builtin.copy:
+    src: "mynetworks"
+    dest: "{{ postfix_allowed_relay_hosts }}"
+    owner: root
+    group: root
+    mode: '0644'
+  when: running_on_server
+
+- name: Postfix | Configure Postfix as relay host
+  ansible.builtin.template:
+    src: main_cf.j2
+    dest: "{{ postfix_main_cf }}"
+    owner: root
+    group: root
+    mode: "0644"
+
+- name: Postfix | Add Proofpoint authentication credentials
+  ansible.builtin.copy:
+    dest: /etc/postfix/sasl_passwd
+    content: |
+      [{{ postfix_smtp_relay_host }}]:{{ postfix_smtp_relay_port }} {{ postfix_smtp_relay_username }}:{{ postfix_smtp_relay_password }}
+    mode: '0644'
```

Comment:
I'm guessing there's a reason, but I don't know what it is, so . . . why are we setting this file to `0644` in one task and then immediately resetting it to `0600` in the next task?